src/Zynga/Framework/PgData/V1/Testing/Mock/PgModel.hh

```diff
@@ -0,0 +1,46 @@
+<?hh //strict
+
+namespace Zynga\Framework\PgData\V1\Testing\Mock;
+
+use Zynga\Framework\PgData\V1\Interfaces\PgModel\ReaderInterface;
+use Zynga\Framework\PgData\V1\Interfaces\PgModel\StatsInterface;
+use Zynga\Framework\PgData\V1\Interfaces\PgModel\WriterInterface;
+use Zynga\Framework\PgData\V1\Interfaces\PgRowInterface;
+use Zynga\Framework\PgData\V1\Testing\Mock\PgReaderWriter;
+use Zynga\Framework\PgData\V1\PgModel as Base;
+
+class PgModel extends Base {
+  private PgReaderWriter $_readerWriter;
```

Comment:
Should this model have a way of defining when to use the PgReaderWriter? If we want to use 'Mock' databases, wont this have to be null ?

Reply:
With the changes to `src/Zynga/Framework/Factory/V2/Base.hh`, when we want to use the Mock databases, we can use the actual PGModel. So we only need to use this model when we want to explicitly use PgReaderWriter.

Reply:
Can you add class doc for this as well